delphi-frontend/src/main/java/au/com/integradev/delphi/symbol/resolve/NameResolver.java

```diff
@@ -852,15 +853,22 @@ private void handleArgumentList(ArgumentListNode node) {
     if (handleExplicitArrayConstructorInvocation(node)) {
       return;
     }
-    disambiguateArguments(node.getArguments(), true);
+
+    disambiguateArguments(
```

Comment:
Could be worth renaming `disambiguateArguments` to `disambiguateArgumentExpressions`?

Reply:
I'd rather not - it lengthens the name but (imo) doesn't buy us much in terms of understanding/clarity.

Reply:
If you're happy with it, I'm not bothered. It's hardly the most confusing thing about the name resolver 😄 

My thoughts were just that since we now have an `ArgumentNode`, `disambiguateArguments` could be expected to disambiguate `ArgumentNode`s rather than `ExpressionNode`s (e.g. width and decimal as well as the core expression).